openwisp_monitoring/monitoring/exceptions.py

```diff
@@ -1,2 +1,14 @@
+from influxdb.exceptions import InfluxDBClientError
+
+from .settings import TIMESERIES_DB
+
+
 class InvalidChartConfigException(Exception):
     pass
+
+
+database_exception = {
+    'influxdb': InfluxDBClientError,
+}
+
+TIMESERIES_DB_EXCEPTION = database_exception[TIMESERIES_DB['DATABASE']]
```

Comment:
This is not the way to define Exceptions in python. This is just a string.

Reply:
@PabloCastellano, I have changed it and I think it's better now but can you please enlighten me on why it's a string (I am not aware).

Reply:
OH you are rgiht, I didn't pay enough attention.